tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml

```diff
@@ -18,6 +18,7 @@ namespace.monitoring:
       inputs:
         mandatory:
           -  TendrlContext.integration_id
+          -  TendrlContext.cluster_name
```

Comment:
Add the new input parameter here too https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/monitoring_utils.py#L9

Reply:
Ah yes while testing I had it here. Forgot to post in PR somehow. Will update the PR

Reply:
Added as part of https://github.com/Tendrl/commons/pull/720